app/javascript/components/App.js

```diff
@@ -19,18 +17,56 @@ export class App extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      shlves: mockShlves,
-      blogs: mockBlogs,
+      shlves: [],
+      blogs: [],
     };
   }
 
-  createBlog = (blog) => {
-    console.log(blog);
-  };
+  componentDidMount(){
+    this.readBlog(),
+    this.readShlf()
+  }
+
+  readBlog(){
+    fetch("http://localhost:3000/blogs")
```

Comment:
create a config.js file - so you can create const and import it.

It'll make it easier to update later.